db/models/User.js

```diff
@@ -18,6 +18,14 @@ const UserSchema = new mongoose.Schema({
         type: String,
         required: true,
         uppercase: true,
+    },
+    resetToken: {
```

Comment:
why we need this in model. Jwt token have expiration in his body.

Reply:
and also jwt need to be part of url. For example http://localhost:3001/reset-password?jwt={token}&userId={userId}

and frontend developer than have token with all information and userId that he need.